packages/gatsby-theme-docs/utils/api-shortcode.js

```diff
@@ -0,0 +1,38 @@
+// loaded via require because we use the pre-ESM version at the moment.
+// to be changed once upgraded to a latest version
+const visit = require('unist-util-visit');
+
+function extractShortcodeOccurrence(shortcodeList, ast) {
+  if (!Array.isArray(shortcodeList) || shortcodeList.length === 0) {
+    return;
+  }
+
+  const shortcodes = []; // {name: shortCodeName, attributes: [{name: foo, value: bar}]}
+
+  const isShortcode = (node) => {
+    return (
+      node.type === 'mdxJsxFlowElement' && shortcodeList.includes(node.name)
+    );
+  };
+
+  // https://unifiedjs.com/explore/package/unist-util-is/#test
```

Comment:
I think this comment line is obsolete.  I originally used unist-util-is but then did it as the custom function in lines 12-16